### DISCUSSION TITLE
Which country is this?

### DISCUSSION THREAD

**Discussion by sxp2015** [wagtail/wagtail#9527] [General]
Hi, Dear Wagtail Development Engineer

I'm from China and I like this software very much, but found this problem, what is this place? Which country is this?

![Snipaste_2022-10-28_15-00-18](https://user-images.githubusercontent.com/10334767/198524164-92daaf1a-44a2-4324-8cbf-c6ea6af31b6d.jpg)

**Comment by gasman** [comment]
Hi @sxp2015,
We don't manage this list - it comes from the `pytz` package - but according to https://en.wikipedia.org/wiki/List_of_tz_database_time_zones it is indeed meant to be Chita, a city in eastern Russia.

**Reply by laymonage** [reply]
Just to add on this: [the format is `Area/Location`](https://en.wikipedia.org/wiki/Tz_database#Names_of_time_zones), where:

- `Area` is "the name of a continent, an ocean, or `Etc`"
- `Location` is "the name of a specific location within the area – usually a city or small island".

So it's not `Area/Country`. For China, you can likely use `Asia/Shanghai`.

**Reply by sxp2015** [reply]
> Hi @sxp2015, We don't manage this list - it comes from the `pytz` package - but according to https://en.wikipedia.org/wiki/List_of_tz_database_time_zones it is indeed meant to be Chita, a city in eastern Russia.

Thank you very much for your reply

**Reply by sxp2015** [reply]
> Just to add on this: [the format is `Area/Location`](https://en.wikipedia.org/wiki/Tz_database#Names_of_time_zones), where:
> 
> * `Area` is "the name of a continent, an ocean, or `Etc`"
> * `Location` is "the name of a specific location within the area – usually a city or small island".
> 
> So it's not `Area/Country`. For China, you can likely use `Asia/Shanghai`.

Thank you very much for your guidance
